tests/functional/z_cluster/nodes/test_non_ocs_taint_and_toleration.py

```diff
@@ -138,19 +150,22 @@ def test_non_ocs_taint_and_tolerations(self):
             logger.info(f"Successfully added toleration to {sub}")
 
         if not config.ENV_DATA["mcg_only_deployment"]:
-            logger.info("Add tolerations to the ocsinitializations.ocs.openshift.io")
-            param = (
-                '{"spec":  {"tolerations": '
-                '[{"effect": "NoSchedule", "key": "xyz", "operator": "Equal", '
-                '"value": "true"}]}}'
-            )
-            ocsini_obj = ocp.OCP(
-                resource_name=constants.OCSINIT,
-                namespace=config.ENV_DATA["cluster_namespace"],
-                kind=constants.OCSINITIALIZATION,
-            )
-            ocsini_obj.patch(params=param, format_type="merge")
-            logger.info(f"Successfully added toleration to {ocsini_obj.kind}")
+            if version.get_semantic_ocs_version_from_config() < version.VERSION_4_15:
```

Comment:
We don't need this condition as we are working with release branches. Just backport the fix till ODF 4.15. The current master will become release branch for 4.16.

Reply:
Then the approach should be to make a branch out of release-4.15 and not from master, and then push the changes.